plans/integration.fmf

```diff
@@ -0,0 +1,25 @@
+summary: Test integration with latest versions of content
+discover+:
+  filter: tag:integration
+prepare:
+  - name: Install packages require for generation ARF files
+    how: install
+    package:
+      - cmake
+      - make
+      - openscap-utils
+      - openscap-scanner
+      - python3-pyyaml
+      - python3-jinja2
+      - python3-setuptools
+      - git
+      - scap-security-guide
+  - name: Generate ARF files
+    how: shell
+    script: 
+      - ./generate_arf.sh ssg no fedora ${TMT_PLAN_DATA}/arf.xml
```

Comment:
Can we parametrize the product?

Reply:
Or it should be marked as only executable in Fedora.

Reply:
FMF must somehow provide the name of the platform is it trying to execute the test on. We might need to map FMF name to our product name, though.